types/office-js-preview/index.d.ts

```diff
@@ -71436,6 +71512,14 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.9]
              */
             rotation?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Specifies the share link to an Office Script file on OneDrive that will be associated with this shape.
+             *
+             * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+             * @beta
+             */
+            scriptLink?: boolean;
```

Comment:
@wandyezj, Why is Script stuff showing up in the public d.ts?

Reply:
This is an Add-In API. It should be ok for this API to be in beta.

Reply:
Removed, resolving conversation.